armi/context.py

```diff
@@ -15,10 +15,7 @@
 """
 Module containing global constants that reflect the executing context of ARMI.
 
-This contains information about the circumstatces under which an ARMI application is
-running. Things like the MPI environment, executing user, etc. live here. These are
-re-exported by the `armi` package, but live here so that import loops won't lead to
-as many issues.
+Things like the MPI environment, executing user, etc. live here.
```

Comment:
Better name than `things`? Maybe "Implements global state attributes that are dependent on the environment, including serial and distributed memory parallelism, Operating System flags, modes of running the framework (e.g., interactive, batch), temporary storage of application files, etc."?

Reply:
Done!